Tests/Units/Translation/TranslationManagerTest.php

```diff
@@ -56,6 +56,10 @@ public function shouldCreateProject()
         $projectMock->expects($this->once())
             ->method('save')
             ->willReturn($projectMock);
+        $projectMock->expects($this->once())
+            ->method('getPotentialAuthors')
+            ->with('my_textmaster')
+            ->willReturn(['55c3763e656462000b000027']);
```

Comment:
According to the [php doc](https://github.com/worldia/textmaster-api/blob/64c5340a839f1bd49f3c463458d51b8198bb4ea6/lib/Textmaster/Model/ProjectInterface.php#L283) the method `getPotentialAuthors` will not return an array of string.